test/toolbox_image.sh

```diff
@@ -193,7 +193,7 @@ cleanup()
 
 run_test()
 {
-    setup
+    setup &> /dev/null
```

Comment:
while I generally approve (even part of this pull request)

for the future, maybe we need to add a variable at the top of our scripts
`UM_DEBUG="${UM_DEBUG:-false}"`

and then do things as:
```sh
SETUP_OUTPUT="&> /dev/null"
if "${UM_DEBUG}"; then
SETUP_OUTPUT=
fi
or something to make it configurable. Reason is, so that we can do pipeline reruns with these variables set/unset allowing us to debug steps on the CI end more easily ...